src/test/java/org/elasticsearch/count/simple/SimpleCountTests.java

```diff
@@ -177,4 +179,46 @@ public void localDependentDateTests() throws Exception {
             assertHitCount(countResponse, 20l);
         }
     }
+
+    @Test
+    public void testThatNonEpochDatesCanBeSearch() throws Exception {
```

Comment:
Does this really need to be an integration test? Could it be in `SimpleDateMappingTests`?